front/app/services/invites.ts

```diff
@@ -81,17 +92,39 @@ export function invitesStream(streamParams: IStreamParams | null = null) {
 }
 
 export async function bulkInviteEmails(object: INewBulkInviteEmails) {
-  const response = await streams.add<IInvites>(
-    `${API_PATH}/invites/bulk_create`,
+  const response = await streams.add(`${API_PATH}/invites/bulk_create`, {
+    invites: object,
+  });
+  await streams.fetchAllWith({ apiEndpoint: [`${API_PATH}/invites`] });
+  queryClient.invalidateQueries({ queryKey: seatsKeys.items() });
+  return response;
+}
+
+export async function bulkInviteXLSX(object: INewBulkXLSXInviteXLSX) {
```

Comment:
Does it make sense for us to move this to `react-query` already or maybe out of scope

Reply:
It does make sense, but my 2 new functions copy the 2 old functions, so we should move them together. And then there will be only `deleteInvite` left in this file, so probably we should move it too. And altogether it looks like quite some work.

Added it to the "simplify" ticket [here](https://citizenlab.atlassian.net/browse/CL-3269).